virt/logging_events_monitoring/virt-reviewing-vm-dashboard.adoc

```diff
@@ -0,0 +1,27 @@
+[id="virt-reviewing-vm-dashboard"]
+= Reviewing resource usage by virtual machines
+include::modules/virt-document-attributes.adoc[]
+:context: virt-reviewing-vm-dashboard
+
+toc::[]
+
+Dashboards in the {ProductName} web console provide visual representations of cluster metrics to help you to quickly understand the state of your cluster. Dashboards belong to the xref:../../monitoring/understanding-the-monitoring-stack.adoc#understanding-the-monitoring-stack[monitoring stack] that provides monitoring for core platform components.
+
+The {VirtProductName} dashboard provides data on resource consumption for virtual machines and associated pods. The dashboard is based on the Grafana analytics platform that enables analysis and visualization of metrics for virtual machines. The visualization metrics displayed in the {VirtProductName} dashboard are based on xref:../../virt/logging_events_monitoring/virt-prometheus-queries.adoc#virt-prometheus-queries[Prometheus Query Language (PromQL) queries].
```

Comment:
@lmandavi  I  think we should drop this sentence:
"The dashboard is based on the Grafana analytics platform that enables analysis and visualization of metrics for virtual machines."
If you decide to keep it so I think its should say
"The dashboard is was created using Grafana visualization and is querying the internal Prometheus server using Prometheus Query Language (PromQL)"